source/tyr/tyr/resources.py

```diff
@@ -987,6 +969,43 @@ def delete(self, user_id, version=0):
         return {}, 204
 
 
+class UserV1(User):
+    def _get_args(self, user_id):
+        parser = super(UserV1, self)._get_args(user_id)
+        parser.add_argument('page', type=int, required=False, default=1)
+        return parser
+
+    def _get_all_users(self, args):
+        del args['disable_geojson']
+        filter_params = {k: v for k, v in args.items() if v and k != 'page'}
+
+        pagination = models.User.query.filter_by(**filter_params).paginate(
+            args['page'], current_app.config.get('MAX_ITEMS_PER_PAGE', 5)
+        )
+        pagination_json = {
+            'current_page': pagination.page,
+            'items_per_page': pagination.per_page,
+            'total_items': pagination.total,
+        }
+        if pagination.has_next:
+            pagination_json['next'] = url_for(request.endpoint, page=pagination.next_num)
+        return marshal(pagination.items, user_fields), pagination_json
+
+    def get(self, user_id=None):
+        resp = super(UserV1, self).get(user_id)
+        if type(resp) == tuple:
```

Comment:
I am a little lost... When is it a tuple ?

Reply:
That's what's returned above (l.992) if there are several users and it needs pagination. The response received is a tuple of the users list and the pagination object.

Reply:
Hoooo, I was looking in the method get from User.... My bad. Could you add a little comment `# If we need pagination` somewhere around here ? Could help blind people like me :D